src/Client/ConfigWindow.java

```diff
@@ -2398,6 +2533,10 @@ public void actionPerformed(ActionEvent e) {
     notificationPanelUseSystemNotifsCheckbox.setToolTipText(
         "Uses built-in system notifications. Enable this to attempt to use your operating system's notification system instead of the built-in pop-up");
 
+    if (SystemTray.isSupported()) {
```

Comment:
just always add it, would be better

Reply:
The affected setting is only shown when `SystemTray.isSupported` (line 2525), so this is wrapped behind the same logic.